src/main/java/seedu/address/model/eatery/Tag.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.model.eatery;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.HashMap;
+
+/**
+ * Represents an Eatery's tag in the EatMe application.
+ */
+public class Tag {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Tags should only contain alphabets and it should not be blank";
```

Comment:
restrict to alphabets and not alphanumeric?

Reply:
yeah..... why would u tag food places with numbers